src/daemon/config.js

```diff
@@ -184,8 +192,13 @@ async function checkPorts (ipfsd) {
   const apiPort = parseInt(configApiMa.nodeAddress().port, 10)
   const gatewayPort = parseInt(configGatewayMa.nodeAddress().port, 10)
 
-  const freeGatewayPort = await portfinder.getPortPromise({ port: gatewayPort, stopPort: gatewayPort + 100 })
-  const freeApiPort = await portfinder.getPortPromise({ port: apiPort, stopPort: apiPort + 100 })
+  const findFreePort = async (port, from) => {
+    port = Math.max(port, from, 1024)
```

Comment:
Hello, this `Math.max` call prevents me from setting a gateway port lower than 8080, and an api port lower than 5001. When I edit the config file and restart ipfs-desktop, it says the port is busy and suggests a new one. 

For example if I try to set gateway port to 6437, then

```js
const gatewayPort = 6437 // read from config file
const freeGatewayPort = await findFreePort(gatewayPort , 8080) // returns 8080

// evaluates to true, even though 6437 is free.
const busyGatewayPort = gatewayPort !== freeGatewayPort 

// I get a popup saying "The port 6437 is not available. Do you want to use 8080 instead?"
```